Content/Bosses/SquidBoss/NPCs.ArenaActor.cs

```diff
@@ -205,12 +213,14 @@ public override void AI()
 					if (Item.type == ItemType<SquidBossSpawn>() && WaterLevel == 150 && !Main.npc.Any(n => n.active && n.ModNPC is SquidBoss)) //ready to spawn another squid              
 					{
 						// Synced spawn, TODO, abstract this to a general synced spawn later?
-						if (Main.netMode == NetmodeID.MultiplayerClient)
+						if (Main.netMode == NetmodeID.Server)
 						{
-							var packet = new SpawnNPC(Main.myPlayer, (int)NPC.Center.X, (int)NPC.Center.Y + 630, NPCType<SquidBoss>());
+							Mod.Logger.Info("Sending packet to spawn auroracle");
+							var packet = new SpawnNPC(-1, (int)NPC.Center.X, (int)NPC.Center.Y + 630, NPCType<SquidBoss>());
```

Comment:
What's happening here? server doesn't need to send this packet to clients. On reviewing the SpawnNPCPacket we have I think that's only useful for a client initiated spawn packet (and even then I don't think that extra Terraria.ID.MessageID.SyncNPC is needed since NPC.NewNPC should send an initial packet anyway) this section looks unnessesary since the block below inside of Main.netMode != NetmodeID.MultiplayerClient should be suficient